### PR TITLE
Add Edit/Delete buttons to JT Schedule Details

### LINKED ISSUE #6171: Add Edit/Delete buttons to JT Schedule Details
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
Link to mockups: https://tower-mockups.testing.ansible.com/patternfly/templates/templates-schedules-detail/

Decided to peel this out of https://github.com/ansible/awx/pull/6170 to make for smaller PRs

### PR DESCRIPTION
**Pull request by nixocio**
Also, add unit-tests to the related changes.

Closes: https://github.com/ansible/awx/issues/6171

UI

schedules_edit_delete

### CODE REVIEW

File: awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.jsx [hunk 1]
```diff
@@ -194,6 +214,39 @@ function ScheduleDetail({ schedule, i18n }) {
           </>
         )}
       </DetailList>
+      <CardActionsRow>
+        {summary_fields.user_capabilities &&
```
Comment (jlmitch5) [inline awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.jsx]: you can make this `summary_fields?.user_capabilities?.edit && ...` instead

File: awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.jsx [hunk 2]
```diff
@@ -194,6 +214,39 @@ function ScheduleDetail({ schedule, i18n }) {
...
+          summary_fields.user_capabilities.edit && (
+            <Button
+              aria-label={i18n._(t`Edit`)}
+              component={Link}
+              to={pathname.replace("details", "edit")}
+            >
+              {i18n._(t`Edit`)}
+            </Button>
+          )}
+        {summary_fields.user_capabilities &&
```
Comment (jlmitch5) [inline awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.jsx]: same with ?.

File: awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.test.jsx [hunk 3]
```diff
@@ -260,4 +262,88 @@ describe('<ScheduleDetail />', () => {
...
+    });
+    await waitForElement(
+      wrapper,
+      'Modal[title="Error!"]',
+      el => el.length === 0
+    );
+    expect(SchedulesAPI.destroy).toHaveBeenCalledTimes(1);
+  });
+
 });
```
Comment (jlmitch5) [inline awx/ui_next/src/components/Schedule/ScheduleDetail/ScheduleDetail.test.jsx:350]: great tests!

**Review by jlmitch5 — COMMENTED** [review]
small tweaks to use ?. and run prettier the next time you commit (that caused the CI to fail) and this should be good!

**Review by jlmitch5 — APPROVED** [review]
nice

**Comment by dsesami** [comment]
Currently cannot fully verify this as it is partially blocked by https://github.com/ansible/awx/issues/6881. Once that closes, I'll make a pass at this PR.

edit: fixed issue link

**Review by dsesami — APPROVED** [review]
Works great! Thanks for this.

**Pull request merged**